### PR TITLE
Add merchant backend

### PR DESCRIPTION
**Pull request by MihailDV**

### CODE REVIEW

**Comment by MihailDV** [comment]
:up:

File: config/settings.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+test:
+  <<: *defaults
+
+production:
+  <<: *defaults
+  checkout_url: "<%= ENV['BG_CHECKOUT_URL'] || 'https://checkout.begateway.com' %>"
+  shop:
+    id: "<%= ENV['BG_SHOP_ID'] || '361' %>"
+    key: "<%= ENV['BG_SHOP_KEY'] || 'b8647b68898b084b836474ed8d61ffe117c9a01168d867f24953b776ddcb134d' %>"
```
Comment (shiroginne) [inline config/settings.yml]: no newline

**Comment by shiroginne** [comment]
:+1: :shipit:

**Comment by MihailDV** [comment]
:up:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  - **description** - order description;
+  - **tracking_id** - optional. the id of your transaction or order.
+
+* **settings** - section has next keys:
+
+  - **success_url** - URL to return Customer to if the transaction was successful;
+  - **decline_url** - URL to return Customer to if the transaction was declined;
+  - **fail_url** - URL to return Customer to if the transaction failed. You can query its status using the `token` and review response status and message parameters to evaluate why the transaction failed;
+  - **cancel_url** - optional. URL to return Customer to if Customer cancels the transaction. The param is in use only if Merchant uses hosted payment page solution;
+  - **notification_url** - optional. URL where notification about a transaction will be posted to;
```
Comment (avsh) [inline README.md]: notification url should be built by the app and posted to ctp. otherwise the endpoint  /tokens/:token is useless.

Comment (MihailDV) [inline README.md]: Could you provide full message for **notification_url**?

**Pull request merged**